browser/layout/wrapper.html

```diff
@@ -102,6 +102,9 @@
 				{{{json this}}}
 			</script>
 		{{/each}}
+		{{#if @root.flags.moatAdsTraffic}}
+			<script async id="moat-ivt" src="https://sejs.moatads.com/financialtimesprebidheader859796398452/yi.js"></script>
```

Comment:
I noticed that this is still being requested even with the `ads` flag off. I think it should probably make sure the ads flag is also on

Reply:
@leggsimon Cheers for flagging, we're going to remove this flag shortly, just waiting on some tings.